int16 lw_speed
int16 rw_speed
uint8 vaccum
uint8 lbrush
uint8 rbrush
uint8 mbrush
uint8 beep
uint8 sleep
bool charge
uint8 ledr
uint8 ledg
bool gyro_state
bool gyro_cali
